client/js/table-view.js

```diff
@@ -82,11 +70,29 @@ class TableView {
 		.forEach(th => this.headerRowEl.appendChild(th));
 	}
 
+	renderTableFooter(){
+		removeChildren(this.footerRowEl);
+		for(var column=0; column <this.model.numCols; column++){
```

Comment:
Replace all "var" with "let" or "const", since ES6 language spec eliminated the need to use "var". For loops use "let".